js/modules/k6/k6_test.go

```diff
@@ -111,12 +111,11 @@ func TestSleep(t *testing.T) {
 			assert.NoError(t, err)
 			dch <- endTime.Sub(startTime)
 		}()
-		runtime.Gosched()
+
 		time.Sleep(1 * time.Second)
-		runtime.Gosched()
```

Comment:
I wonder why these were here? :confused: Seems they were here from almost the start (https://github.com/grafana/k6/commit/274eef82cbafef77a9eb8461e7a3d920bfd875e7), but they don't seem necessary, so :+1: for removing them